kubernetes/client/api/admissionregistration_v1alpha1_api.py

```diff
@@ -3,41 +3,39 @@
 """
     Kubernetes
 
-    No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+    No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)  # noqa: E501
 
-    OpenAPI spec version: v1.13.1
-    
-    Generated by: https://github.com/swagger-api/swagger-codegen.git
+    OpenAPI spec version: v1.13.3
+    Generated by: https://openapi-generator.tech
 """
 
 
 from __future__ import absolute_import
 
-import sys
-import os
-import re
+import re  # noqa: F401
```

Comment:
There are a lot of changes which clean the code (unused imports, long lines etc.).